submitted_by.module

```diff
@@ -190,12 +190,15 @@ function submitted_by_comment_view($comment, $view_mode, $langcode) {
 
 
 /**
- * Implements hook_process_node().
+ * Implements hook_preprocess_node().
  *
- * Use hook_process_node() rather than hook_preprocess_node() as themes might
- * override the submitted variable unconditionally.
+ * The Drupal 7 module uses hook_process_node() rather
+ * than hook_preprocess_node() as themes might
+ * override the submitted variable unconditionally; however,
+ * hook_process_node() does not exist in Backdrop, so we use
+ * hook_preprocess_node() instead.
```

Comment:
...nit-picking, I know, but there's some funny line wrapping going on there. That block should be like so instead (wrap at 80 chars):

```
/**
  * Implements hook_preprocess_node().
  *
  * The Drupal 7 module uses hook_process_node() rather than
  * hook_preprocess_node() as themes might override the submitted variable
  * unconditionally; however, hook_process_node() does not exist in Backdrop, so
  * we use hook_preprocess_node() instead.
  */
```